easybuild/easyconfigs/m/MultiQC/MultiQC-1.7-foss-2018b-Python-3.6.6.eb

```diff
@@ -95,7 +95,12 @@ exts_list = [
     }),
     ('multiqc', version, {
         'source_urls': ['https://pypi.python.org/packages/source/m/multiqc/'],
-        'checksums': ['02e6a7fac7cd9ed036dcc6c92b8f8bcacbd28983ba6be53afb35e08868bd2d68'],
+        'patches': ['%(namelower)s-%(version)s_allow-matplotlib-3-for-python-3.patch'],
+        'checksums': [
+            '02e6a7fac7cd9ed036dcc6c92b8f8bcacbd28983ba6be53afb35e08868bd2d68',  # multiqc-1.7.tar.gz
+            # multiqc-1.7_allow-matplotlib-3-for-python-3.patch
+            '75bf2978fdb147a306c9db6b230681958a57cc7b67d2c2f50cbaa208ecb6ee0c',
```

Comment:
@SethosII Checksum for patch file should be `672724742be15a886e756973b6fb1567210c9137c6a062e29f1802929a0f88fb`?

Reply:
@boegel Mh, had CR LF from windows in the file. I changed it.